lib/libmbedtls/core/ecc.c

```diff
@@ -379,3 +342,76 @@ TEE_Result crypto_acipher_ecc_shared_secret(struct ecc_keypair *private_key,
 	mbedtls_ecdh_free(&ecdh);
 	return res;
 }
+
+static const struct crypto_ecc_keypair_ops ecc_keypair_ops = {
+	.generate = &ecc_generate_keypair,
+	.sign = &ecc_sign,
+	.shared_secret = &ecc_shared_secret,
+};
+
+TEE_Result crypto_asym_alloc_ecc_keypair(struct ecc_keypair *s,
+					 uint32_t key_type,
+					 size_t key_size_bits)
+{
+	switch (key_type) {
```

Comment:
Well, I guess it will be more complex to read than help because:
parameter `s` is not the same and so `ops` is not the same.
`key_type` value is not the same.

I would keep it as is.